tools/dev_cluster.py

```diff
@@ -75,6 +81,45 @@ class NodeConfig:
     cluster_size: int
 
 
+class Minio:
+    def __init__(self, binary, directory):
+        self.binary = binary
+        self.directory = directory
+        self.stopped = False
+
+    def stop(self):
+        if not self.stopped:
+            self.stopped = True
+            self.process.send_signal(signal.SIGINT)
+
+    async def run(self):
+        log_path = self.directory / "minio.log"
+        data_dir = self.directory / "data"
+        #conf_dir = self.directory / "config"
+        home_dir = self.directory / "home"
+        data_dir.mkdir(parents=True, exist_ok=True)
+        #conf_dir.mkdir(parents=True, exist_ok=True)
+        home_dir.mkdir(parents=True, exist_ok=True)
+
+        env = dict(HOME=home_dir,
+                   MINIO_REGION_NAME="panda-region",
+                   MINIO_DOMAIN="localhost")
```

Comment:
Doesn't this meed to be the values from Redpanda config?

Reply:
yeh, fixed